PowerPointLabs/PowerPointLabs/TextCollection.cs

```diff
@@ -18,161 +18,197 @@ public class TextCollection
         # region Ribbon XML
         # region Supertips
         # region Animation Lab
+        public const string AnimationLabMenuSupertip =
+            "Use Animation Lab to add animations your slides easily.";
         public const string AddAnimationButtonSupertip =
-            "Creates an animation slide to transition from the currently selected slide to the next slide.";
-        public const string ReloadButtonSupertip =
-            "Recreates an existing animation slide with new animations.\n\n" +
-            "To activate, select the original slide or the animation slide then click this button.";
+            "Creates an animation slide to transition from the currently selected slide to the next slide.\n\n" +

```

Comment:
"Creates" should be changed to "Create". Also for Remove Audio, Add Captions, Remove Captions, Remove Highlighting, Generate Audio Automatically.